tests/e2e/conftest.py

```diff
@@ -0,0 +1,109 @@
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+
+"""Pytest configuration."""
+
+from __future__ import absolute_import, print_function
+
+import os
+import sys
+
+import pytest
+from flask import Flask
+from flask_babelex import Babel
+from flask_cli import FlaskCLI
+from flask_mail import Mail
+from flask_menu import Menu
+from invenio_db import InvenioDB, db
+from selenium import webdriver
+
+from invenio_accounts import InvenioAccounts
+from invenio_accounts.views import blueprint
+
+
+@pytest.fixture()
+def app(request):
+    """Flask application fixture for E2E/integration/selenium tests.
+
+    Overrides the `app` fixture found in `../conftest.py`. Tests/files in
+    this folder and subfolders will see this variant of the `app` fixture.
+    """
+    app = Flask('testapp_e2e')
+    app.config.update(
+        ACCOUNTS_USE_CELERY=False,
+        CELERY_ALWAYS_EAGER=True,
+        CELERY_CACHE_BACKEND="memory",
+        CELERY_EAGER_PROPAGATES_EXCEPTIONS=True,
+        CELERY_RESULT_BACKEND="cache",
+        MAIL_SUPPRESS_SEND=True,
+        SECRET_KEY="CHANGE_ME",
+        SECURITY_PASSWORD_SALT="CHANGE_ME_ALSO",
+        SQLALCHEMY_DATABASE_URI=os.getenv('SQLALCHEMY_DATABASE_URI',
+                                          'sqlite://'),
+        TESTING=True,
+        LOGIN_DISABLED=False,
+    )
+    FlaskCLI(app)
+    Menu(app)
+    Babel(app)
+    Mail(app)
+    InvenioDB(app)
+    InvenioAccounts(app)
+    app.register_blueprint(blueprint)
+
+    with app.app_context():
+        db.create_all()
+
+    def teardown():
+        with app.app_context():
+            db.drop_all()
+
+    request.addfinalizer(teardown)
+    return app
+
+
+def pytest_generate_tests(metafunc):
+    """Overrides pytest's default test collection function.
+    For each test in this directory which uses the `env_browser` fixture,
+    said test is called once for each value found in the
+    `E2E_WEBDRIVER_BROWSERS` environment variable."""
+    if 'env_browser' in metafunc.fixturenames:
+        browsers = os.environ.get('E2E_WEBDRIVER_BROWSERS',
+                                  failobj='Firefox').split()
+        metafunc.parametrize('env_browser', browsers, indirect=True)
+
+
+@pytest.fixture()
+def env_browser(request):
+    """Fixture for a webdriver instance of the browser specified by the
+    `WEBDRIVER_BROWSER` environment variable."""
+
+    # Create instance of webdriver.`request.param`()
+    browser = getattr(webdriver, request.param)()
+    # If the test fails it's be useful to know which browser was used.
+    print("Webdriver browser=", request.param)
```

Comment:
using `print` is not the best way to to detect fail - you should have this information in traceback context.